From 4e39281c1f380c53ee6ef8b9795111e0695a55be Mon Sep 17 00:00:00 2001
From: Elliott Hughes <enh@google.com>
Date: Tue, 10 Nov 2015 18:34:38 -0800
Subject: [PATCH] Document %s in date(1).

---
 toys/posix/date.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/toys/posix/date.c b/toys/posix/date.c
index 398b891..ca0ca53 100644
--- a/toys/posix/date.c
+++ b/toys/posix/date.c
@@ -35,6 +35,7 @@ config DATE
     %a short weekday name    %A weekday name         %u day of week (1-7, 1=mon)
     %b short month name      %B month name           %Z timezone name
     %j day of year (001-366) %d day of month (01-31) %e day of month ( 1-31)
+    %s seconds past the Epoch
 
     %U Week of year (0-53 start sunday)   %W Week of year (0-53 start monday)
     %V Week of year (1-53 start monday, week < 4 days not part of this year) 
